__tests__/integration.test.js

```diff
@@ -157,6 +157,26 @@ describe("/api/articles/:article_id", () => {
     })
   })
 
+  test.only("GET: Returned articles should now additionaly have a comment_count property", () => {
+    return request(app)
+    .get("/api/articles/4")
+    .expect(200)
+    .then(({body})=>{
+      const {article} = body
+      expect(article).toMatchObject({
+        article_id: 4,
+        title: expect.any(String),
+        topic: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number),
+        article_img_url:expect.any(String),
+        comment_count: 0
+      })
```

Comment:
No need to test again for all the other properties, our old test proves we get those back fine. We only need to assert against the new comment_count property here and both tests should pass

Reply:
Good to know! That did occur to me but I thought I'd keep them in to be safe.